test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -770,6 +770,90 @@ class test_ucp_sockaddr : public ucp_test {
         return false;
     }
 
+    void do_force_close_during_rndv(bool fail_send_ep)
+    {
+        constexpr size_t length = 4 * UCS_KBYTE;
+
+        listen_and_communicate(false, SEND_DIRECTION_BIDI);
+
+        mem_buffer recv_buffer(length, UCS_MEMORY_TYPE_HOST);
+        recv_buffer.pattern_fill(2, length);
+        void *rreq = recv(receiver(), recv_buffer.ptr(), length,
+                          rtag_complete_check_data_cbx,
+                          reinterpret_cast<void*>(&recv_buffer));
+
+        mem_buffer send_buffer(length, UCS_MEMORY_TYPE_HOST);
+        send_buffer.pattern_fill(1, length);
+        void *sreq = send(sender(), send_buffer.ptr(), length,
+                          send_recv_type(), scomplete_reset_data_cbx,
+                          reinterpret_cast<void*>(&send_buffer));
+
+        ucp_ep_h ep = sender().revoke_ep();
+
+        // Wait for the TAG RNDV/RTS packet sent and the request scheduled to
+        // be tracked until RNDV/ATS packet is not received from a peer
+        ucs_time_t deadline = ucs::get_deadline();
+        do {
+            ASSERT_LT(ucs_get_time(), deadline);
+            ucp_worker_progress(sender().worker());
+        } while (ucs_hlist_is_empty(&ucp_ep_ext_gen(ep)->proto_reqs));
+        EXPECT_FALSE(ucs_hlist_is_empty(&ucp_ep_ext_gen(ep)->proto_reqs));
```

Comment:
maybe wait for send req completion?

Reply:
> maybe wait for send req completion?

we don't need to wait for send_req completion here, we want to wait for sreq being scheduled on tracking list and wait for ATS

Reply:
can we post send, progress few times to put it on track list - but just do not post recv yet?

Reply:
> can we post send, progress few times to put it on track list - but just do not post recv yet?

implemented it by waiting for the message came to receiver by using `tag_probe` and do `tag_msg_recv` for the message